.github/workflows/release.yaml

```
@@ -12,19 +12,64 @@ env:
+      artifactory_password: ${{ secrets.TRUEFOUNDRY_ARTIFACTORY_PUBLIC_PASSWORD }}
+
+  # Release the Helm chart
   release:
     runs-on: ubuntu-latest
+    needs: [build-operator, build-resolver]
     steps:
       - name: Checkout main
         uses: actions/checkout@v4
         with:
           fetch-depth: "0"
     
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)